src/Resources/config/doctrine_phpcr_form_types.xml

```diff
@@ -10,8 +10,11 @@
         <service id="sonata.admin.doctrine_phpcr.form.type.phpcr_odm_tree_manager" class="Sonata\DoctrinePHPCRAdminBundle\Form\Type\TreeManagerType">
             <tag name="form.type" alias="doctrine_phpcr_odm_tree_manager"/>
         </service>
-        <service id="sonata.admin.doctrine_phpcr.form.collection_type_extension" class="Sonata\DoctrinePHPCRAdminBundle\Form\Extension\CollectionTypeExtension" public="true">
+        <service id="sonata.admin.doctrine_phpcr.form.collection_type_extension_legacy" class="Sonata\DoctrinePHPCRAdminBundle\Form\Extension\DeprecatedCollectionTypeExtension" public="true">
```

Comment:
Please, add a "NEXT_MAJOR" comment explaining how to proceed about these changes on the next major release.